components/FeaturedCarousel.tsx

```diff
@@ -13,17 +13,22 @@ import {
 import { ChevronLeftIcon, ChevronRightIcon } from "lucide-react";
 import Image from "next/image";
 import Icon from "@/components/ui/render-icon";
+import Markdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw'
+import remarkGfm from 'remark-gfm'
 
 export interface FeaturedCarouselItem {
   title: string;
-  category: string;
+  category: string | string[];
```

Comment:
This would be easier as jus a `string[]`

I'd suggest doing something like 
```
categories: string[]
```

so then you're not accepting multiple types